src/main/java/it/pagopa/pm/gateway/service/VposDeleteService.java

```diff
@@ -38,21 +37,22 @@ public class VposDeleteService {
     @Value("${vpos.requestUrl}")
     private String vposUrl;
```

Comment:
Put this field too as a constructor parameter

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/199/commits/a0177d35a94e09792e1dc9940fdd20f35934ddbc